### ISSUE TITLE
Cannot import ArtifactoryPath

### ISSUE THREAD

**Issue by saurabhbhagwat** [devopshq/artifactory#105]
ImportError: cannot import name 'ArtifactoryPath'

Getting this error when trying to import. Please let me know the solution ASAP

**Comment by Tim55667757** [comment]
@saurabhbhagwat what version of dohq-artifactory do you used?
Check version please (latest release is 0.5.241):
$ pip show dohq-artifactory

Name: dohq-artifactory
Version: 0.5.241
Summary: A Python interface to Artifactory
Home-page: https://devopshq.github.io/artifactory/
Author: Alexey Burov
Author-email: aburov@ptsecurity.com
License: MIT License
Location: c:\anaconda3\lib\site-packages
Requires: python-dateutil, requests
Required-by: crosspm, artifactory-du

**Comment by nikolasj** [comment]
@saurabhbhagwat Do you get an error using `from artifactory import ArtifactoryPath`?

**Comment by ardyflora** [comment]
@nikolasj I might be wrong but it looks I might have similar issue.

Error
> onx-rflora-01:scripts $ python auto_artifactory.py
Traceback (most recent call last):
  File "auto_artifactory.py", line 1, in <module>
    from artifactory import ArtifactoryPath
ImportError: No module named artifactory


Code line:
``
from artifactory import ArtifactoryPath
``

Tried to follow example mentioned in the documentation:
[https://github.com/devopshq/artifactory#walking-directory-tree](url)

The output from the command:
``
pip show dohq-artifactory
``

> onx-rflora-01:scripts $ pip show dohq-artifactory
Name: dohq-artifactory
Version: 0.5.241
Summary: A Python interface to Artifactory
Home-page: https://devopshq.github.io/artifactory/
Author: Alexey Burov
Author-email: aburov@ptsecurity.com
License: MIT License
Location: /usr/lib/python3.6/site-packages
Requires: requests, python-dateutil
Required-by:

**Comment by nikolasj** [comment]
@ardyflora `python auto_artifactory.py` what version of python do you used? maybe 2.7?
try it 
```python3 -mpip install -U dohq-artifactory
python3 auto_artifactory.py```

**Comment by ardyflora** [comment]
@nikolasj 
Python version:

> python --version
Python 2.7.5

Dohq-artifactory requirements looks good
> onx-rflora-01:~ $ python3 -mpip install -U dohq-artifactory
Requirement already up-to-date: dohq-artifactory in /usr/lib/python3.6/site-packages (0.5.241)
Requirement already satisfied, skipping upgrade: python-dateutil in /usr/lib/python3.6/site-packages (from dohq-artifactory) (2.8.0)
Requirement already satisfied, skipping upgrade: requests in /usr/lib/python3.6/site-packages (from dohq-artifactory) (2.22.0)
Requirement already satisfied, skipping upgrade: six>=1.5 in ./.local/lib/python3.6/site-packages (from python-dateutil->dohq-artifactory) (1.12.0)
Requirement already satisfied, skipping upgrade: certifi>=2017.4.17 in /usr/lib/python3.6/site-packages (from requests->dohq-artifactory) (2019.9.11)
Requirement already satisfied, skipping upgrade: idna<2.9,>=2.5 in /usr/lib/python3.6/site-packages (from requests->dohq-artifactory) (2.8)
Requirement already satisfied, skipping upgrade: chardet<3.1.0,>=3.0.2 in /usr/lib/python3.6/site-packages (from requests->dohq-artifactory) (3.0.4)
Requirement already satisfied, skipping upgrade: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/lib/python3.6/site-packages (from requests->dohq-artifactory) (1.25.6)

But when I run the following:
``python3` auto_artifactory.py
``
Error:

> onx-rflora-01:~ $ python3 auto_artifactory.py
python3: can't open file 'auto_artifactory.py': [Errno 2] No such file or directory

I didn't clone the project because I was under the impression all I have to do is ``pip install`` 
`` import artifactoryPath`` and start using it. Please correct me if I am wrong.

**Comment by ardyflora** [comment]
Should I open this as a new issue?

**Comment by alsam1992** [comment]
@ardyflora 
> onx-rflora-01:~ $ python3 auto_artifactory.py
python3: can't open file 'auto_artifactory.py': [Errno 2] No such file or directory

here is another mistake.
do the following:
```
# python3 -mpip install -U dohq-artifactory
# python3
# from artifactory import ArtifactoryPath
```
send a mistake.

that's what i have
```
# python3
Python 3.6.8 (default, Oct  7 2019, 12:59:55) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 
>>> from artifactory import ArtifactoryPath
>>>
```
I have no mistake

**Comment by ardyflora** [comment]
Thank you, I see what I did wrong.

**Comment by nikolasj** [comment]
@ardyflora @saurabhbhagwat close the issue?

**Comment by ardyflora** [comment]
Yes please, Thank you for the help.
